saleor/graphql/core/scalars.py

```diff
@@ -36,6 +36,24 @@ def parse_value(value):
             return None
 
 
+class PositiveDecimal(Decimal):
+    """Positive Decimal scalar implementation.
+
+    Should be used in places where value must be positive.
+    """
+
+    @staticmethod
+    def parse_value(value):
+        value = super(PositiveDecimal, PositiveDecimal).parse_value(value)
```

Comment:
wouldn't `value = super().parse_value(value)` do the same?

Reply:
No, if it's a static method you must do it in this way.